events.js

```diff
@@ -95,8 +118,61 @@ EventEmitter.init = function() {
   }
 
   this._maxListeners = this._maxListeners || undefined;
+
+  if (opts && opts.captureRejections) {
+    if (typeof opts.captureRejections !== 'boolean') {
+      throw new TypeError('The "options.captureRejections" argument must be of type boolean. Received type ' + typeof opts.captureRejections);
+    }
+    this[kCapture] = Boolean(opts.captureRejections);
+  } else {
+    this[kCapture] = EventEmitter.prototype[kCapture];
+  }
 };
 
+var ProcessNextTick = typeof queueMicrotask === 'function'
+  ? queueMicrotask
+  : typeof setImmediate === 'function'
+  ? setImmediate
+  : setTimeout;
```

Comment:
This should use `process.nextTick` instead so it has the same shared queue behaviour as in node.js. Would be a breaking change for webpack 5 users who might not be polyfilling the `process` global yet.